src/Microsoft.Bcl.HashCode/tests/Microsoft.Bcl.HashCode.Tests.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
```

Comment:
We only use two whitespaces for nesting in msbuild files. Update both the source project and this one. See System.Buffers as an example.